### ISSUE TITLE
js error in 4.3.1 build

### ISSUE THREAD

**Issue by tylorngo** [olton/Metro-UI-CSS#1425]
Got this issue in IE in 4.3.1 build. I switched back to 4.3.0 and it works fine.

![image](https://user-images.githubusercontent.com/17262969/66395765-db4e7480-e99d-11e9-94b5-f510b47476c3.png)

**Comment by olton** [comment]
https://github.com/olton/Metro-UI-CSS/blob/master/CONTRIBUTING.md#issues-guidelines

**Comment by olton** [comment]
Thank you for your feedback. For IE11 and less require next code before metro (fucking IE not support property name for the function):
```
if (!(function f() {}).name) {
    Object.defineProperty(Function.prototype, 'name', {
        get: function() {
            var name = (this.toString().match(/^function\s*([^\s(]+)/) || [])[1];
            Object.defineProperty(this, 'name', { value: name });
            return name;
        }
    });
}
``` 
I will add this code in 4.3.2 to m4q (1.0.2)

**Comment by olton** [comment]
Fixed in 4.3.2
